app/controllers/users_controller.rb

```diff
@@ -213,6 +212,8 @@ def profile
           flash.now[:warning] = I18n.t('users_controller.user_has_been_moderated')
         end
       end
+      @count_ans = @profile_user.answers.count
+      @count_ques = @profile_user.questions.where(status: 1).length
```

Comment:
You can remove both of them and can use instance variables defined above in this method. Ref: https://github.com/publiclab/plots2/pull/5310#discussion_r270660582